adapters/ipmi_generic/apply_errors.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/*
+ * Version: $Id$
+ * Created: Jun 8, 2012
+ */
+
+// Error strings to detect failure in configuration application
+$apply_errors = array(
+  "@exceeds max-dn@",
```

Comment:
probably this list can be cleaned up, for instance max-dn is from ipphone (I think)